test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go

```diff
@@ -403,11 +471,7 @@ func kubeletRestartCmd() []string {
 }
 
 func createPod(ctx context.Context, c client.Client, ns string, opts ...func(pod *corev1.Pod)) (*corev1.Pod, error) {
-	p := pods.GetTestPod()
-	p.Namespace = ns
-	for _, opt := range opts {
-		opt(p)
-	}
+	p := createPodTemplate(ctx, c, ns, opts...)
```

Comment:
Why do we need another function here?

Reply:
The thing was that the previous createPod function not only constructed the pod template it also created it on the cluster by sending a request to the API server. I wanted to decouple these two functions because I need only the first part of this (creating the template) for the deployment.
I could create this function separately and not change anything in the original `createPod` function but this would result in some code duplication.